.bazelrc

```diff
@@ -0,0 +1,11 @@
+common --cxxopt="-std=c++20"
+common --host_cxxopt="-std=c++20"
+
+# Ensure we are using a specific macOS SDK even if our Xcode version has a newer SDK
+common --macos_minimum_os=14.0
+
+# Only use Bzlmod
+common --noenable_workspace
+
+# Put your custom flags in `./user.bazelrc`, which is gitignored
+try-import user.bazelrc
```

Comment:
Could we consolidate this with the `.bazelrc`s in [spoor/.build](https://github.com/microsoft/spoor/tree/master/.build)?

I don't believe that try-import was available when we first wrote the project so I like your idea of putting the `.bazelrc` in the root directory.

We'll also want to update CI: https://github.com/microsoft/spoor/blob/master/.github/workflows/build-and-test.yml#L33-L34